utils/config/daos_server.yml

```diff
@@ -215,6 +215,13 @@
 #firmware_helper_log_file: /tmp/daos_firmware.log
 #
 #
+## Enable HTTP endpoint for remote telemetry collection.
+#
+## default endpoint state: disabled
+## default endpoint port: 9191
+#telemetry_port: 9191
```

Comment:
I am a little concerned about landing this without going through unit tests. There are some control plane tests that use the example config to validate its config generation code.

I recommend using a commit pragma that will ensure that the unit test stage is run. 
e.g. 
Skip-func-test: true
Skip-nlt: true

There may be a better choice but that one at least should guarantee that unit tests are run while bypassing all of the rest of the testing.

Reply:
aaah, i keep forgetting that some of our workflow is "un-marshalling" the commented-out parts of the YML file, so it isn't truly all-comments....

Reply:
done - added WithTelemetryPort(9191) to the server unit tests